cmd/restic/progress.go

```diff
@@ -110,6 +110,10 @@ func newIndexTerminalProgress(quiet bool, json bool, term *termstatus.Terminal)
 	return newTerminalProgressMax(!quiet && !json && stdoutIsTerminal(), 0, "index files loaded", term)
 }
 
+func newVerifyTerminalProgress(quiet bool, json bool, term *termstatus.Terminal) *progress.Counter {
```

Comment:
no need for a wrapper function here. Just call `newTerminalProgressMax` directly from `cmd_restore`.